ga/src/main/java/nl/tudelft/serg/evosql/sql/parser/SqlSecurerVisitor.java

```diff
@@ -172,18 +172,6 @@ public void visit(Parenthesis arg0) {
 
 	@Override
 	public void visit(StringValue arg0) {
```

Comment:
you can remove this method too, right?

Reply:
It's an overridden method from the `ExpressionVisitor` interface. Method itself can't be removed.

Reply:
ahh, from an interface! got it!